trackData/frontend.json

```diff
@@ -4,7 +4,7 @@
     "category": "A",
     "description": "Develops expertise in web client technologies, such as HTML, CSS, and JavaScript",
     "milestones": [{
-        "points": 1.5,
```

Comment:
those .5 fractions caused some values to appear at the boundaries between roles which resulted in an empty list of titles

Reply:
Good point! +1